util/shared_lib.py

```diff
@@ -1,4 +1,5 @@
-from gpu import get_available_gpus
+from __future__ import print_function
+from .gpu import get_available_gpus
```

Comment:
This file needs to import `__future__.absolute_import` as well, right? Also, could you change this to `from util.gpu` instead of `.gpu`?